watertap/examples/flowsheets/electrodialysis/electrodialysis_1stack_conc_recirc_ui.py

```diff
@@ -0,0 +1,634 @@
+        name="Cell width",
+        ui_units=pyunits.meter,
+        display_units="m",
+        rounding=3,
+        description="The width of ED cell or stack",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+    exports.add(
+        obj=fs.EDstack.cell_length,
+        name="Channel length",
+        ui_units=pyunits.meter,
+        display_units="m",
+        rounding=3,
+        description="The length of ED cell or stack",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+
+    exports.add(
+        obj=fs.EDstack.spacer_porosity,
+        name="Spacer porosity",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Porosity of the flow spacer",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.spacer_porosity,
+        name="Spacer porosity",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Porosity of the flow spacer",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.spacer_specific_area,
+        name="Spacer specific surface area",
+        ui_units=pyunits.meter**-1,
+        display_units="m^-1",
+        rounding=0,
+        description="Specific surface area of the flow spacer",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.electrodes_resistance,
+        name="Electrode resistance",
+        ui_units=pyunits.ohm * pyunits.meter**2,
+        display_units="ohm m^2",
+        rounding=2,
+        description="Areal resistance of the two electrodes",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.EDstack.current_utilization,
+        name="Current utilization coefficient",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Current utilization coefficient",
+        is_input=True,
+        input_category="ED stack",
+        is_output=False,
+    )
+
+    exports.add(
+        obj=fs.recovery_vol_H2O,
+        name="Water recovery by volume",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Product water recovery by volume",
+        is_input=True,
+        input_category="ED stack",
+        is_output=True,
+    )
+
+    # Membrane-related properties
+    exports.add(
+        obj=fs.EDstack.water_trans_number_membrane["cem"],
+        name="Water electroosmosis transport number of CEM",
+        ui_units=pyunits.dimensionless,
+        display_units="1",
+        rounding=2,
+        description="Water electroosmosis transport number of CEM",
```

Comment:
Spell out CEM here and in similar descriptions below? The description is what pops up when a user hovers over a variable, so it should clear up any acronyms that we don't expect users to be familiar with